src/version_req.rs

```diff
@@ -84,13 +90,13 @@ enum WildcardVersion {
 
 #[derive(Clone, PartialEq, Eq, PartialOrd, Ord, Hash, Debug)]
 enum Op {
-    Ex, // Exact
-    Gt, // Greater than
-    GtEq, // Greater than or equal to
-    Lt, // Less than
-    LtEq, // Less than or equal to
-    Tilde, // e.g. ~1.0.0
-    Compatible, // compatible by definition of semver, indicated by ^
+    Ex,                        // Exact
```

Comment:
I thought doing this was non-standard style; I generally prefer not aligning them. I haven't reviewed the relevant Libz Blitz guidelines though

Reply:
I don't think we really cover formatting in the guidelines, but personally also prefer them not aligned.